test/apps_finance.js

```diff
@@ -0,0 +1,265 @@
+const { assertInvalidOpcode } = require('./helpers/assertThrow')
+const { getBlockNumber } = require('./helpers/web3')
+
+const Vault = artifacts.require('Vault')
+const FinanceApp = artifacts.require('FinanceAppMock')
+const MiniMeToken = artifacts.require('MiniMeToken')
+
+contract('Finance App', accounts => {
+    let app, vault, token1, token2, executionTarget = {}
+
+    const n = '0x00'
+    const periodDuration = 100
+    const maxUint = 0
+
+    beforeEach(async () => {
+        vault = await Vault.new()
+
+        token1 = await MiniMeToken.new(n, n, 0, 'n', 0, 'n', true)
```

Comment:
What's this? It's a lot of mental mapping, so it's hard to understand. Can we be clearer here? The ability to read tests is very very crucial since many contributors might look here instead of documentation (I usually do).

Reply:
I agree that test legibility could be very much improved, but that case it is just initializing the token with empty parameters :)